script/.bashrc

```diff
@@ -1,20 +1,12 @@
+#!/bin/bash
+. "$ASDF_HOME/.asdf/asdf.sh"
 
-. /home/ubuntu/.asdf/asdf.sh
+. "$ASDF_HOME/.asdf/completions/asdf.bash"
 
-. /home/ubuntu/.asdf/completions/asdf.bash
+#to add erlang to PATH
+export PATH=$ASDF_HOME/.asdf/installs/erlang/20.0/bin:$PATH
```

Comment:
I think adding elixir and erlang to PATH would be a better way for shell access to their commands.

Reply:
asdf.bash is supposed to produce the path completion affect for elixir and erlang. Once configured correctly, the required commands come into the PATH.

Reply:
https://github.com/VedangW/TransportScheduler/blob/1cc7099d84e76d19fefc1a5dfb66432ee5216b00/script/.bashrc#L6-L19

Then the aliases can be removed from the .bashrc file as asdf will handle adding elixir and erlang to PATH.

Reply:
My bad. I think aliases are the ones that actually enable us to enter commands like erl. Now, I am not sure of the work done by asdf.bash. Please look into the contents of the file.

Reply:
asdf.bash is a shell script wrapper for asdf itself. We need that also.